src/stcal/jump/jump.py

```diff
@@ -250,6 +282,11 @@ def detect_jumps(frames_per_group, data, gdq, pdq, err,
             previous_row_above_gdq = row_above_gdq.copy()
             k += 1
         elapsed = time.time() - start
+        if expand_large_events:
```

Comment:
Might want to just add a comment in the code here that this is the call that flags snowballs

Reply:
added